src/main/kotlin/com/soberg/aoc/utlities/datastructures/Grid2D.kt

```diff
@@ -21,6 +21,8 @@ data class Grid2D<T>(
 
     infix operator fun get(location: Location): T = grid[location.row][location.col]
 
+    fun get(row: Int, col: Int): T = grid[row][col]
```

Comment:
FYI this could be an operator fun!

```suggestion
    operator fun get(row: Int, col: Int): T = grid[row][col]
```

Reply:
Usage would be:
```kotlin
val row = 5
val col = 3
val value = grid[row, col]
```